aQute.libg/test/aQute/lib/io/IOTest.java

```diff
@@ -297,8 +322,10 @@ public void testCreateSymlinkOrCopyWillDeleteOriginalLink() throws Exception {
 		}
 	}
 
-	public void testCreateDirectory_Symlink() throws Exception {
-		Path rootDirectory = Paths.get("generated/tmp/test/" + getName());
+	@Test
+	@DisabledOnOs(WINDOWS)
```

Comment:
Why did you disable this on Windows? This test currently runs on Windows.

Reply:
My bad. It didn't work on my box because I hadn't configured siblings to work properly. Easy to renable